<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow_dll</class>
 <widget class="QDialog" name="userWindow_dll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="transactionButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TRANSACTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balanceButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BALANCE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TRANSFER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>WITHDRAW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="depositButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DEPOSIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="fname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Veli-Matti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditWithdrawBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CREDIT WITHDRAW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditPaymentBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CREDIT PAYMENT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hakala</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SPEDEN PANKKI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fname_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
